config/locales/en/en_admin.yml

```diff
@@ -0,0 +1,35 @@
+en:
```

Comment:
The comment is for the filename `config/locales/en/en_admin.yml` why you have duplicated the `en` in it?
`config/locales/en/admin.yml` looks more suitable here.